src/sparseml/core/logger/logger.py

```diff
@@ -948,49 +959,162 @@ def log_scalar(
         level: Optional[int] = None,
     ):
         """
+        (Note: this method is deprecated and will be removed in a future version,
+        use LoggerManager().metric.log_scalar instead)
+
         :param tag: identifying tag to log the value with
         :param value: value to save
         :param step: global step for when the value was taken
         :param wall_time: global wall time for when the value was taken
         :param kwargs: additional logging arguments to support Python and custom loggers
         :return: True if logged, False otherwise.
         """
-        for log in self.loggers:
-            if log.enabled and (log_types == ALL_TOKEN or log.name in log_types):
-                log.log_scalar(
-                    tag=tag,
-                    value=value,
-                    step=step,
-                    wall_time=wall_time,
-                    level=level,
-                )
+        warnings.warn(
```

Comment:
I think we should just remove the warnings to avoid confusion on older integrations, since this isn't something that will be updated in the integrations